modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/IncidentContactCard/IncidentContactCard.jsx

```diff
@@ -0,0 +1,291 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import {useEffect, useState} from 'react';
+import {getHighPriorityContacts} from '~/common/services/liferay/api';
+import i18n from '../../../../common/I18n';
+
+export const HIGH_PRIORITY_CONTACT_CATEGORIES = {
+	criticalIncident: i18n.translate('critical-incident'),
+	privacyBreach: i18n.translate('privacy-breach'),
+	securityBreach: i18n.translate('security-breach'),
+};
+
+const IncidentContactCard = ({accountSubscriptionGroupsNames}) => {
+	const incidentContactStandart = 2;
```

Comment:
`incidentContactStandart` looks like a spelling error? is `Standart` supposed to be `Standard`?

Reply:
Done!